src/Exception/ExceptionCollection.php

```diff
@@ -15,9 +15,16 @@
  * A collection of exceptions that might occur during the flush operation of a
  * ProxyClientInterface implementation
  */
-class ExceptionCollection extends \Exception implements \IteratorAggregate, \Countable,  HttpCacheExceptionInterface
+class ExceptionCollection extends \Exception implements \IteratorAggregate, \Countable, HttpCacheExceptionInterface
 {
     private $exceptions = array();
+    
+    public function __construct(array $exceptions = array())
+    {
+        foreach ($exceptions as $exception) {
+            $this->add($exception);
```

Comment:
should the add method become private? kind of unrelated to this PR but when we start refactoring for 2.0 anyways...